doc/user/evpn.rst

```diff
@@ -502,3 +502,11 @@ the creation of the VLAN subinterfaces and directly enslave "eth0" to "br10".
       ip link set eth0 master br10
 
 This completes the necessary configuration for an L2VNI.
+
+Displaying evpn mac vni information
```

Comment:
cometic nit: EVPN, MAC, VNI keywords just look better all-capital :shrug:

Reply:
And if this is a section, it might be a good place not only for MAC VNI commands?

Reply:
Fixed it